src/ProgressOnderwijsUtils/SchemaReflection/CheckConstraint.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using Microsoft.Data.SqlClient;
+using static ProgressOnderwijsUtils.SafeSql;
+
+namespace ProgressOnderwijsUtils.SchemaReflection
+{
+    public struct CheckConstraintEntry : IWrittenImplicitly

```

Comment:
ik heb hier weer een struct van gemaakt. De readpoco ging er op stuk omdat die een parameterloze constructor verwacht:
![image](https://user-images.githubusercontent.com/17001435/106138206-1add3900-616c-11eb-9e6a-a34796e9a7ba.png)

Reply:
pas readpocos aan; dit eis is niet langer redelijk.